### ISSUE TITLE
Disabling Legend makes X-axis labels leave component bounds

### ISSUE THREAD

**Issue by xxfogs** [AppDevNext/AndroidChart#261]
When chart legend gets disabled through

```kt
chart.legend.isEnabled = false
```

X-axis labels become just partially visible due to leaving the component bounds

![image](https://github.com/user-attachments/assets/b1811c66-d7b1-433a-bad4-ad00b317693a)

I believe this issue has been inherited from the upstream as a [similar issue has been filed recently](https://github.com/PhilJay/MPAndroidChart/issues/5462)

**Comment by hannesa2** [comment]
Sure, as far as I know, this legend part is untouched.
The only diff is here, that I merge pull requests and create immediate a new release

**Comment by xxfogs** [comment]
Do you maybe know any workarounds till a patch is released?

**Comment by hannesa2** [comment]
No.
To solve this systematically, a new espresso test, which demonstrates the issue is required.

**Comment by hannesa2** [comment]
I tried to reproduce it https://github.com/AppDevNext/AndroidChart/pull/262 but here it's fine (at least for me)
Please can you provide a running sample, best in this repo as a new sample ?

**Comment by xxfogs** [comment]
Yes, I will provide a sample repo, however can you please try moving the X axis in your test to bottom (`chart.xAxis.position = XAxis.XAxisPosition.BOTTOM`)? Currently it's on top

### Test:
![image](https://github.com/user-attachments/assets/303ecf4c-a5dc-4baa-992f-1dad21882a32)

### My sample:
![image](https://github.com/user-attachments/assets/48d86474-9bf5-40b6-8021-710558ff1da2)

**Comment by hannesa2** [comment]
> Yes, I will provide a sample repo,

Brtter, to do it in this repo as a pull request. Or simply jump into https://github.com/AppDevNext/AndroidChart/pull/262
